### PR TITLE
Add URLHaus API

### PR DESCRIPTION
**Pull request by FH089**
- [x] My submission is formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [ ] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ API | Description | Auth | HTTPS | CORS |
 | [AlienVault Open Threat Exchange (OTX)](https://otx.alienvault.com/api) | IP/domain/URL reputation | `apiKey` | Yes | Unknown |
 | [Google Safe Browsing](https://developers.google.com/safe-browsing/) | Google Link/Domain Flagging | `apiKey` | Yes | Unknown |
 | [Metacert](https://metacert.com/) | Metacert Link Flagging | `apiKey` | Yes | Unknown |
+| [URLhaus](https://urlhaus-api.abuse.ch/)| Bulk queries and Download Malware Samples| No | Yes | Unknown|
```
Comment (matheusfelipeog) [inline README.md:102]: Each segment must start and end with exactly 1 space.

```suggestion
| [URLhaus](https://urlhaus-api.abuse.ch/) | Bulk queries and Download Malware Samples | No | Yes | Unknown |
```

Can you fix this, please?

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @FH089 ✋ 

The API is Ok. Only the markdown format is organized incorrectly.

Can you solve this review, please?

**Review by matheusfelipeog — APPROVED** [review]
Thanks @FH089 😄

**Comment by FH089** [comment]
your welcome @matheusfelipeog  :+1:

**Pull request merged**